beefy-gadget/src/lib.rs

```diff
@@ -32,9 +32,12 @@ use sp_runtime::traits::Block;
 
 use beefy_primitives::BeefyApi;
 
+use crate::validator::BeefyGossipValidator;
```

Comment:
```suggestion
use validator::BeefyGossipValidator;
```

Reply:
Or even better remove this and just use `validator::BeefyGossipValidator::new()`